Traceback (most recent call last):
  File "/Users/orcanon/Desktop/DI725/assignment_1/train.py", line 322, in <module>
    train_dataset = SentimentDataset('train')
  File "/Users/orcanon/Desktop/DI725/assignment_1/train.py", line 130, in __init__
    self.df = pd.read_csv('./train.csv')  # or wherever your raw CSV is
  File "/Users/orcanon/.conda/envs/pythonProject3/lib/python3.10/site-packages/pandas/util/_decorators.py", line 211, in wrapper
    return func(*args, **kwargs)
  File "/Users/orcanon/.conda/envs/pythonProject3/lib/python3.10/site-packages/pandas/util/_decorators.py", line 331, in wrapper
    return func(*args, **kwargs)
  File "/Users/orcanon/.conda/envs/pythonProject3/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 950, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/Users/orcanon/.conda/envs/pythonProject3/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 605, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/Users/orcanon/.conda/envs/pythonProject3/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 1442, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/Users/orcanon/.conda/envs/pythonProject3/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 1735, in _make_engine
    self.handles = get_handle(
  File "/Users/orcanon/.conda/envs/pythonProject3/lib/python3.10/site-packages/pandas/io/common.py", line 856, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: './train.csv'
